common/config/processor.go

```diff
@@ -0,0 +1,66 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/odigos-io/odigos/common"
+	"sigs.k8s.io/controller-runtime/pkg/log"
```

Comment:
I am not sure how this logger will work outside of a controller runtime k8s environment. we should consider returning the error/s back to the caller to log or write into the CRD as status conditions.

Reply:
Check 196c64ff2b0efb3d9c425abccdb4269e8a22d149.